lib/themes/reps/css/reps.css

```diff
@@ -1558,9 +1555,6 @@ ul.pager li.pager-current {
 .view-filters .views-exposed-widgets .views-widget-filter-keys {
   width: 40%;
 }
-.view-filters .views-exposed-widgets .views-widget-filter-combine {
```

Comment:
Keep this line for correct display on the news filtering since we changed the way the filter works.